docs/SUMMARY.md

```diff
@@ -1,5 +1,6 @@
 # Summary
 
+* [←](https://www.chartjs.org)
```

Comment:
I think `←` might be slightly confusing because it looks like the back button and chartjs.org might not be the last page you visited.

A couple other options:
- Rename the existing "Chart.js" link to "Introduction". Add a new link that says something like "Chart.js" or "Homepage"
- Add a link to the samples instead. That's the only other thing that's on the homepage. It'd be fewer clicks that way because you could go straight to the samples instead of navigating through the homepage

Reply:
True, you are right. I think both of your options make sense. If you somehow end up on docs without having been on the homepage before then the first option would make more sense. What do you think?